ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/validation/validation.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+import { InvalidSearchFormMessage } from '../../../component/announcement/CommonMessages'
+const announcement = require('../../../component/announcement/index.jsx')
+
+export function validate(errors: any) {
+  if (errors.length == 0) {
+    return true
+  }
+  let searchErrorMessage = JSON.parse(JSON.stringify(InvalidSearchFormMessage))
+  if (errors.length > 1) {
+    let msg = searchErrorMessage.message
+    searchErrorMessage.title =
+      'Your Search Cannot Be Run due to Multiple Errors'
```

Comment:
This capitalization seems inconsistent here
```suggestion
      'Your search cannot be run due to multiple errors'
```

Reply:
done 👍